README.md

```diff
@@ -771,28 +777,81 @@ let numbers = create([Number], [1, 2, 3, 4]);
 </ul>;
 ```
 
-# Observable Microstates
+# Streaming State
 
-By themselves microstates are purely functional. They have no builtin concept of identity,
-time or sequencing. However, it is precisely because of these properties that they can be
-used with almost any state management solution that does. In fact, Microstates comes bundled
-out of the box with the ability to convert any microstate into a stream of transitions using
-the the Observable API.
+A microstate represents a single immutable value and provides transitions to derive the next value. Microstates provides a mechanism called Identity that allows to emit stream of Microstates. When you create an Identity from a Microstate, you get an object that has the same shape the original microstate. Every composed microstate becomes an identity and every transition gets wrapped in side effect emitting behaviour specific to the identity's constructor. Identity wrapped Microstate offer the following benefits over raw Microstates.
```

Comment:
This seems _way_ more clear to me now. Basically, `Identity` is the original `Microstate` plus the additional logic that allows you to subscribe to the changes over time, either through `Store` or `Observable`. Is that right?

Reply:
Yes